spatialos-sdk/src/worker/connection.rs

```diff
@@ -1,21 +1,18 @@
-use std::ffi::{CStr, CString};
-use std::ptr;
-
-use futures::{Async, Future};
-
-use spatialos_sdk_sys::worker::*;
-
 use crate::ptr::MutPtr;
 use crate::worker::commands::*;
-use crate::worker::component;
-use crate::worker::component::internal::{CommandRequest, CommandResponse, ComponentUpdate};
+use crate::worker::component::internal::ComponentUpdate;
+use crate::worker::component::{self, Component};
 use crate::worker::entity::Entity;
 use crate::worker::internal::utils::cstr_to_string;
 use crate::worker::locator::*;
 use crate::worker::metrics::Metrics;
 use crate::worker::op::OpList;
-use crate::worker::parameters::{CommandParameters, ConnectionParameters};
+use crate::worker::parameters::ConnectionParameters;
 use crate::worker::{EntityId, InterestOverride, LogLevel, RequestId};
+use futures::{Async, Future};
+use spatialos_sdk_sys::worker::*;
+use std::ffi::{CStr, CString, NulError};
+use std::ptr;
```

Comment:
Small nit: Judicious use of white space can organise imports nicely :smile:

Reply:
Hrmmm, this might be a personal taste thing, but I think grouping all the imports together and letting rustfmt sort them is the better approach. rustfmt only seems to sort them if there's no spaces between them, so the only way to make sure imports are organized consistently across all files is to group them up and let rustfmt handle it.

My suggestion to get nicer organization would be to make more use of nested imports, e.g.:

```
use futures::{Async, Future};
use spatialos_sdk_sys::worker::*;
use std::{
    ffi::{CStr, CString, NulError},
    ptr,
};
```

That has the same affect (i.e. grouping related imports together) while still ensuring that rustfmt can fully format imports.

Reply:
That's a shame that `rustfmt` doesn't handle the whitespace well :confused: 

In which case I agree grouping the imports is a good compromise :)